.github/workflows/generate-swagger.yml

```
@@ -0,0 +1,247 @@
+        env:
+          CHAIN_TYPE: ${{ matrix.chain-type != 'default' && matrix.chain-type || '' }}
+
+      - name: Upload OpenAPI spec
+        uses: actions/upload-artifact@v4
+        with:
+          name: openapi-spec-${{ matrix.chain-type }}
+          path: openapi.${{ matrix.chain-type }}.yaml
+          retention-days: 1
+
+  push-specs:
+    needs: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)